scene/theme/default_theme.cpp

```diff
@@ -801,6 +801,7 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_color("parent_hl_line_color", "Tree", Color(0.27, 0.27, 0.27));
 	theme->set_color("children_hl_line_color", "Tree", Color(0.27, 0.27, 0.27));
 	theme->set_color("custom_button_font_highlight", "Tree", control_font_hover_color);
+	theme->set_color("stripe_color", "Tree", Color(1, 1, 1, 0.02));
```

Comment:
This *may* be preferable as a default option, so that you can still see stripes if you change the background to a light color:

```suggestion
	theme->set_color("stripe_color", "Tree", Color(0.5, 0.5, 0.5, 0.04));
```

On the other hand, this will reduce contrast between lines on dark themes that have a low contrast (with a background that's close to gray), so it's not always a win.